modules/repository/repo.go

```diff
@@ -169,7 +173,7 @@ func MigrateRepositoryGitData(ctx context.Context, u *user_model.User,
 		}
 	}
 
-	ctx, committer, err := db.TxContext(db.DefaultContext)
+	subCtx, committer, err := db.TxContext(ctx)
```

Comment:
`ctx, committer, err := db.TxContext(ctx)` could be clearer and avoid abuse the incorrect ctx.

Reply:
I dont' think so. It made me confused. Use different variables make it more clear.

Reply:
If anyone writes something in `db.GetXxx(ctx)` in this block in the future, do you think it can be reviewed and found easily?

Reply:
Done.